board/fischertechnik/TXT/rootfs/bin/calibrate-touchscreen

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+calibrate() {
+	sudo TSLIB_TSDEVICE=/dev/input/event0 /usr/bin/ts_calibrate
+	/sbin/set-touchscreen-calibration-reset-flag
```

Comment:
Now that you've encapsulated the touch/rm, these are _really_ unneccessary. Just use touch directly here.